buildSrc/src/main/java/org/opensearch/gradle/testclusters/RunTask.java

```diff
@@ -74,14 +76,30 @@ public class RunTask extends DefaultTestClustersTask {
 
     @Option(option = "debug-jvm", description = "Enable debugging configuration, to allow attaching a debugger to opensearch.")
```

Comment:
I suggest changing the descriptions to the following:

- `debug-jvm`: Run OpenSearch as a debug client, where it will connect to a debugging server (or fail) at startup
- `debug-server-jvm`: Run OpenSearch as a debug server that will accept connections from a debugging client

Otherwise it's not super clear what the difference is between the options. What do you think?

Reply:
@andrross hm ... the more I reading over the `debug-jvm`, more I am convinced it is supposed to do exactly what the suggested `debug-server-jvm` is doing:  "Enable debugging configuration, to allow attaching a debugger to opensearch." Let me check few things first ...

Reply:
:+1: updated, thanks!

Reply:
Honestly I would be fine with removing the existing one. What is the use case for wanting OpenSearch to connect to a debugger as opposed to the other way around? It's not really a problem to keep both but I'm not sure I'll ever use `debug-jvm` now that this new option exists.

Reply:
> It's not really a problem to keep both but I'm not sure I'll ever use debug-jvm now that this new option exists.

I don't really know to be honest, but I will try to find out, for now I have not figured out the way to make this option useful.